src/common/interpolation.c

```diff
@@ -907,6 +1064,21 @@ void dt_interpolation_compute_pixel4c(const struct dt_interpolation *itor, const
     *(__m128 *)out = _mm_set_ps1(0.0f);
   }
 }
+#endif
+
+void dt_interpolation_compute_pixel4c(const struct dt_interpolation *itor, const float *in, float *out,
+                                      const float x, const float y, const int width, const int height,
+                                      const int linestride)
+{
+  if(darktable.codepath.OPENMP_SIMD)
```

Comment:
TLDR: So i guess the answer is: yes, but it is so to keep it similar with other `*_plain()` functions.

There is a really deep thought behind it..
So there is a function `dt_interpolation_compute_pixel4c()`, which is a virtual, and calls some implementation, i.e. `dt_interpolation_compute_pixel4c_sse()` if there is SSE, or `dt_interpolation_compute_pixel4c_plain()`.
BUT, *_sse() variant has some vectorization, right?
So we can add some #pragma openmp magic into *_plain() variant, to put vectorization back, but automatically, compiler-driven.

Note: this may be wrong for this particular case, but it is the idea for all this virtual functions like this `dt_interpolation_compute_pixel4c()`.